.github/workflows/ci.yaml

```diff
@@ -10,20 +10,25 @@ env:
   DOTTY_CI_RUN: true
 
 jobs:
-  test:
+  test_non_bootstrapped:
     runs-on: [self-hosted, Linux]
     container:
-      image: lampepfl/dotty:2020-09-08
+      image: lampepfl/dotty:2020-11-19
       volumes:
         - ${{ github.workspace }}/../../cache/sbt:/root/.sbt
         - ${{ github.workspace }}/../../cache/ivy:/root/.ivy2/cache
         - ${{ github.workspace }}/../../cache/general:/root/.cache
-    if: "!(github.event_name == 'push' &&
-           startsWith(github.event.ref, 'refs/tags/sbt-dotty-'))"
-
+    if: "github.event_name != 'pull_request'
+         || (
+           !contains(github.event.head_commit.message, '[skip ci]')
+           && (
+             contains(github.event.head_commit.message, '[test_non_bootstrapped]')
+             || (github.event_name == 'push' && !startsWith(github.event.ref, 'refs/tags/sbt-dotty-'))
```

Comment:
You may have unintentionally changed the behavior here.  Previously, the job would be skipped if an sbt-dotty tag was pushed.  Now, it will run for all events other than pull requests.